frontend/src/container/GridCardLayout/WidgetHeader/WidgetHeader.styles.scss

```diff
@@ -2,7 +2,7 @@
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
-	height: 40px;
+	height: 30px;
```

Comment:
Are we sure this doesn't affect other widgets?

Reply:
Tested, and looks good.